fetch.bs

```diff
@@ -805,8 +802,8 @@ A: 3
  <li><p>Otherwise, <a for=list>append</a> (<var>name</var>, <var>value</var>) to <var>list</var>.
 </ol>
 
-<p class="note no-backref"><a for="header list">Combine</a> is used by {{XMLHttpRequest}} and the
-<a lt="establish a WebSocket connection">WebSocket protocol handshake</a>.
+<p class=note><a for="header list">Combine</a> is used by {{XMLHttpRequest}} and the
+<a spec=websockets lt="establish a WebSocket connection">WebSocket protocol handshake</a>.
```

Comment:
This `spec=` shouldn't be necessary, here and below. It pierces the `export=""` barrier so is generally bad.

I guess it is necessary for now since Bikeshed gets confused between Fetch and WebSockets as the link destination. But that's better fixed with link-defaults plus a TODO to remove them in the future, I believe.

Reply:
In the previous PR you argued we should use `spec=` for this one since there was no reason for WebSockets to export it.

Reply:
Oh, I thought you were making them export it. I guess I got confused. It's fine then if you're not.

Reply:
I made WebSockets export obtain a WebSocket connection (that is used by Fetch). Establish a WebSocket connection is somewhat confusingly named but that is the handshake part the API ends up invoking.